main.go

```diff
@@ -22,7 +22,8 @@ import (
 	"github.com/urfave/cli"
 )
 
-const commandHelpTemplate string = `{{.HelpName}}{{if .UsageText}}
+const (
+	commandHelpTemplate = `{{.HelpName}}{{if .UsageText}}
```

Comment:
We don't really need those templates. Looks how elegant is build-info-go with urfave v2: https://github.com/jfrog/build-info-go/blob/main/main.go

Reply:
Looks great! :)